lib/rubocop/ast/node/mixin/method_dispatch_node.rb

```diff
@@ -8,6 +8,8 @@ module MethodDispatchNode
       extend NodePattern::Macros
       include MethodIdentifierPredicates
 
+      ACCESS_MODIFIERS = %i[public protected private module_function].freeze
```

Comment:
True. I was thinking it should only be for the bare method calls since it is used for things like rails indentation behavior. I'll see if it works to add these in